src/app/Http/Controllers/Auth/ResetPasswordController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\ResetPasswordRequest;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Facades\Password;
+
+
+class ResetPasswordController extends Controller
+{
+     // Show the password reset form
+     public function showResetForm(Request $request, $token): View
+     {
+         return view('auth.passwords.reset', ['token' => $token, 'email' => $request->email]);
+     }
+
+     // Reset the user's password
```

Comment:
Use docbloc but don't include the @param or @return on it because you already assigned the types of your parameter and return on your method. Kind a redundant right??


https://docs.phpdoc.org/guide/getting-started/what-is-a-docblock.html